start-gateway-config.sh

```diff
@@ -27,5 +27,6 @@ if [ "$prevent_start" = 1 ]; then
 else
 	# Check dbus container is ready and then launch config
     wait_for_dbus \
-        && python gatewayconfig
+        && python gatewayconfig \
+        && upnpc -e "Nebra Helium" -r 44158 TCP
```

Comment:
I'm not 100% sure because I can't test it on real hardware. On my end my RPi is working but the hm-miner keeps resetting so I can't track the port 44158.

From the code perspective though, it was looking fine to me.

Reply:
@MuratUrsavas @marvinmarnold Actually this would never get executed AFAIK. because `&&` implies that the previous command should exit successfully. If we don't expect `python gatewayconfig` to exit (unless it exits in response to unhandled errors) then there is no scenario where this gets executed.

Reply:
Case in point:

![image](https://user-images.githubusercontent.com/198638/151224211-4fa9223d-ca47-495d-8138-363e5cca9000.png)

Reply:
Good catch @kashifpk - see https://github.com/NebraLtd/hm-config/pull/166